node/src/accountant/scanners/payable_scanner/tx_templates/initial/new.rs

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2025, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+use crate::accountant::db_access_objects::payable_dao::PayableAccount;
+use crate::accountant::scanners::payable_scanner::tx_templates::BaseTxTemplate;
+use std::ops::{Deref, DerefMut};
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct NewTxTemplate {
+    pub base: BaseTxTemplate,
+}
+
+impl From<&PayableAccount> for NewTxTemplate {
+    fn from(payable_account: &PayableAccount) -> Self {
+        Self {
+            base: BaseTxTemplate::from(payable_account),
+        }
+    }
+}
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct NewTxTemplates(pub Vec<NewTxTemplate>);
+
+impl From<Vec<NewTxTemplate>> for NewTxTemplates {
+    fn from(new_tx_template_vec: Vec<NewTxTemplate>) -> Self {
+        Self(new_tx_template_vec)
+    }
+}
+
+impl Deref for NewTxTemplates {
+    type Target = Vec<NewTxTemplate>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl DerefMut for NewTxTemplates {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.0
+    }
+}
+
+impl IntoIterator for NewTxTemplates {
+    type Item = NewTxTemplate;
+    type IntoIter = std::vec::IntoIter<Self::Item>;
+
+    fn into_iter(self) -> Self::IntoIter {
+        self.0.into_iter()
+    }
+}
+
+impl FromIterator<NewTxTemplate> for NewTxTemplates {
+    fn from_iter<I: IntoIterator<Item = NewTxTemplate>>(iter: I) -> Self {
+        NewTxTemplates(iter.into_iter().collect())
+    }
+}
+
+impl From<&Vec<PayableAccount>> for NewTxTemplates {
+    fn from(payable_accounts: &Vec<PayableAccount>) -> Self {
+        Self(
+            payable_accounts
+                .iter()
+                .map(|payable_account| NewTxTemplate::from(payable_account))
+                .collect(),
+        )
+    }
+}
+
```

Comment:
Anyway, having seen those impls in this file, I think something fishy is going on here. This is not how one usually writes code. Probably AI code here. But weird.